app/assets/stylesheets/layouts/_employeeform.scss

```diff
@@ -0,0 +1,141 @@
+.form-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
You should alphabetically order your styles

Reply:
How I can resolve it?

Reply:
Sort them, if you use VSCode you can type `CTRL + P` or `CMD + P` and choose the Sort Lines Ascending. Affect the lines that are select

Reply:
Exactly like German said 
Do it on all the styles is a good practice:)

Reply:
thanks @germanruzca and @juperameza